### PR TITLE
Add missing documentation on Cypher.concat

### PR DESCRIPTION
**Pull request by angrykoala**

### CODE REVIEW

**Review by MacondoExpress — APPROVED** [review]
Looks good!

**Review by lidiazuin — COMMENTED** [review]
Just a few suggestions :) ty!

File: docs/modules/ROOT/pages/how-to/concatenate-clauses.adoc [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[[concatenate-clauses]]
+:description: This page describes how to concatenate clauses.
+= Concatenate Clauses
```
[the remark this answers is not in the document]
Comment (angrykoala) [reply docs/modules/ROOT/pages/how-to/concatenate-clauses.adoc:3]: I was thinking it as a verb "How to concatenate clauses"

[the remark this answers is not in the document]
Comment (lidiazuin) [reply docs/modules/ROOT/pages/how-to/concatenate-clauses.adoc]: In this case, don't you think "concatenating clauses" sounds better? It's possibly just a personal choice, so feel free to ignore me 😅

**Pull request merged**